openapi.yaml

```diff
@@ -0,0 +1,468 @@
+swagger: "2.0"
+info:
+  description: "This is the api description for the SneakerBot by samc621"
+  version: "1.0.0"
+  title: "SneakerBot"
+host: "localhost"
+basePath: "/v1"
+tags:
+- name: "address"
+  description: "Everything about your addresses"
+- name: "proxies"
+  description: "Everything about your proxies"
+- name: "tasks"
+  description: "Everything about your tasks"
+schemes:
+- "https"
+paths:
+  /addresses:
+    post:
+      tags:
+      - "address"
+      summary: "Add a new address"
+      description: ""
+      operationId: "createAddress"
+      consumes:
+      - "application/json"
+      produces:
+      - "application/json"
+      parameters:
+      - in: "body"
+        name: "body"
+        description: "The address object that needs to be added"
+        required: true
+        schema:
+          $ref: "#/definitions/Address"
+      responses:
+        "200":
+          description: "Address successfully created"
+        "405":
```

Comment:
Did you mean 404 here? You can check `/helpers/server-response.js` to see the possible server response codes.

Reply:
Fixed